scene/gui/flow_container.cpp

```diff
@@ -156,17 +163,42 @@ void FlowContainer::_resort() {
 		// but only if the line doesn't contain a child that expands.
 		if (child_idx_in_line == 0 && Math::is_equal_approx(line_data.stretch_ratio_total, 0)) {
 			int alignment_ofs = 0;
+			float prior_stretch_avail = current_line_idx != 0 && !line_data.is_filled ? lines_data[current_line_idx - 1].stretch_avail : 0.0;
```

Comment:
```suggestion
			float prior_stretch_avail = (current_line_idx != 0 && !line_data.is_filled) ? lines_data[current_line_idx - 1].stretch_avail : 0.0;
```
More clear

Reply:
Good catch - I had previously thought to declare a bool there to avoid some repetition in that code block, which I have done in the push below.